Makefile

```diff
@@ -51,6 +51,19 @@ export GO111MODULE
 all:
 	$(MAKE) -C $(dir $(BASE_DIR)) build
 
+.PHONY: lint
+lint:
+	@echo "running golangci-lint"
+	@lintBin=$$(go env GOPATH)/bin/golangci-lint ; \
+	if [ ! -f "$${lintBin}" ]; then \
+		lintBin=$$(echo ./bin/golangci-lint) ; \
+		if [ ! -f "$${lintBin}" ]; then \
+			echo "golangci-lint executable not found" ; \
+			exit 1; \
+		fi \
+	fi ; \
+	$${lintBin} run --new
+
```

Comment:
This is not hooked to any phase yet
should we create a `check` phase, that runs both `common-check-license` and `lint` together? And later on, we can integrate this check into CI/CD pipeline.

Reply:
Yes it will become part of the jenkins job or github PR workflow when we have our full build pipeline up and running and have the Apache Incubator move finalised.